guides/common/modules/proc_synchronizing-a-content-view-to-a-smartproxy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Synchronizing_a_Content_View_to_a_{smart-proxy-context}{context}"]
+= Synchronizing a content view to a {SmartProxy}
+
+You can use the Hammer CLI to synchronize a content view to a {SmartProxy}.
+You need to specify the content view or content view ID along with the lifecycle environment or lifecycle environment ID.
+You will need to include the organization or organization ID as seen in the following examples:
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer capsule content synchronize --id 1 \
+--organization "Default Organization" --content-view-id 42
+# hammer capsule content synchronize --id 1 \
+--organization-id 1 --lifecycle-environment-id 999
+# hammer capsule content synchronize --name \
+"My Capsule" --organization "Default Organization" --content-view "My CV"
+# hammer capsule content synchronize --name \
+"My Capsule" --organization-id 1 --lifecycle-environment "Production"
+----
+
+.CLI procedure
+* Synchronize the content view to a {SmartProxy} by using Hammer:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer capsule content synchronize --content-view "my content view name"
+# hammer capsule content synchronize --content-view-id 42
```

Comment:
```suggestion
$ {hammer-smart-proxy} content synchronize --content-view "my content view name"
```

* I'd drop the second, almost identical example. I don't think we always mention the different between labels, IDs, and names.
* You can run `hammer` as normal user -> Use "$" prefix
* There's an attribute for "hammer capsule": https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L80 & https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L79